packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -171,6 +171,44 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
     })
   })
 
+  context('Runs - Create Project', () => {
+    it('when a project is created, injects new projectId into the config file', () => {
+      cy.remoteGraphQLIntercept(async (obj) => {
+        if (obj.operationName === 'SelectCloudProjectModal_CreateCloudProject_cloudProjectCreate') {
+          obj.result.data!.cloudProjectCreate = {
+            slug: 'newProjectId',
+            id: 'newId',
+          }
+        }
+
+        return obj.result
+      })
+
+      cy.scaffoldProject('launchpad')
+      cy.openProject('launchpad')
+      cy.startAppServer('e2e')
+      cy.loginUser()
+      cy.visitApp()
+
+      cy.withCtx(async (ctx) => {
+        const config = await ctx.project.getConfig()
+
+        expect(config.projectId).to.not.equal('newProjectId')
```

Comment:
would this value be undefined?

Reply:
At this point it's `null`. I checked `to.not.equal('newProjectId')` instead of `to.equal(null)` so that if at some poit in the future a projectId is added to the scaffolded project, this test won't break 😃 happy to change if you prefer

Reply:
> so that if at some point in the future a projectId is added to the scaffolded project

I'm not as familiar with the dashboard space, but if a new scaffolded project had a projectId, are users able to create a new project & override it from the UI?

Reply:
> if a new scaffolded project had a projectId, are users able to create a new project & override it from the UI
I believe so